qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -68,6 +68,16 @@ POOL=mirror
 SRC_DIR=$(readlink -f $(dirname $0)/../../../src)
 TEMPDIR=
 
+# These vars facilitate running this script in an environment with
+# ceph installed from packages, like teuthology. These are not defined
+# by default.
+#
+# USE_EXISTING_CLUSTER - if set, do not start and stop ceph clusters
+# UNMANAGED_RBD_MIRROR - if set, do not start and stop rbd-mirror
+# CEPH_ID              - id portion of client.$id ceph user that rbd-mirror uses
+#                        this is assumed to be the same for all clusters, and is
+#                        only used if UNMANAGED_RBD_MIRROR is set
+
```

Comment:
CEPH_ID is set for all workunits by teuthology; I'll add the prefix to the others